src/environments/array.js

```diff
@@ -125,9 +126,18 @@ function parseArray(
                 hLinesBeforeRow.push([]);
             }
             break;
-        } else if (next === "\\cr") {
-            const cr = assertNodeType(parser.parseFunction(), "cr");
-            rowGaps.push(cr.size);
+        } else if (next === "\\\\") {
+            parser.consume();
+            let size;
+            // \def\Let@{\let\\\math@cr}
+            // \def\math@cr{...\math@cr@}
+            // \def\math@cr@{\new@ifnextchar[\math@cr@@{\math@cr@@[\z@]}}
+            // \def\math@cr@@[#1]{...\math@cr@@@...}
+            // \def\math@cr@@@{\cr}
```

Comment:
Does this collapse multiple `\cr`s in a row or something like that?

Reply:
@kevinbarabash No, they are just how `\\` is defined in LaTeX and amsmath.

Reply:
I don't think I'll ever get used to reading .dtx files. 😓